main.py

```diff
@@ -770,6 +775,12 @@ def clean_up_db(self):
             if len(verification[email]['verified']) == 1 and len(verification[email]['unverified']) >= 1:
                 db.delete(verification[email]['unverified'])
 
+class CronHandler(BaseHandler):
```

Comment:
I feel like it is better to try and be consistent. Also, a lot of the code that is used for certain cron tasks (invoke a url, the parse functions) may be reused by other cron tasks in the future.